virtualcluster/config/crd/cluster.x-k8s.io_clusters.yaml

```diff
@@ -1,6 +1,6 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1
```

Comment:
I think you might need to keep this one based on the spec since v1beta1 and v1 have a breaking change that screws this up.

Reply:
My local test fails with v1 since the apiserver that the test brings up only has crd version v1beta1 supported.

Reply:
I have to copy over the old crd schema to make the prow tests succeed I believe.

Reply:
Ah that is a good note, what version of k8s do you run locally? `apiextensions.k8s.io /v1beta1` is deprecated in `v1.22` so we should make sure we're moving towards that since alpha.2 is out.